docs/tutorials/multimodal/advanced_topics/customization.ipynb

```diff
@@ -360,29 +360,6 @@
     "```\n"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "id": "02d07866",
-   "metadata": {},
-   "source": [
-    "### optimization.track_grad_norm\n",
-    "\n",
-    "Track the p-norm of gradients during training. May be set to ‘inf’ infinity-norm. If using Automatic Mixed Precision (AMP), the gradients will be unscaled before logging them."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "1b60c371",
-   "metadata": {},
-   "source": [
-    "```\n",
-    "# default used by AutoMM (no tracking)\n",
-    "predictor.fit(hyperparameters={\"optimization.track_grad_norm\": -1})\n",
-    "# track the 2-norm\n",
-    "predictor.fit(hyperparameters={\"optimization.track_grad_norm\": 2})\n",
-    "```\n"
-   ]
-  },
```

Comment:
We still support using track_grad_norm , so no need to remove it. See https://github.com/autogluon/autogluon/blob/master/multimodal/src/autogluon/multimodal/optimization/lit_module.py#L401-L404

Reply:
I remember we had some conversation on it in https://github.com/autogluon/autogluon/pull/3190/commits/14e6fee69956d833ad41675ce7cf44bb1b198939 - I'm on holiday, maybe @yinweisu you could take over?

Reply:
Happy holiday! Lightning's trainer has removed the `track_grad_norm` argument. We are using a suggested workaround of doing that.

Reply:
yap. @ddelange Happy holiday and thanks for your contribution!